### ISSUE TITLE
Music

### ISSUE THREAD

**Issue by philiprbrenan** [philiprbrenan/SlipsAndSpills#38]
Please provide a capability to:

(a) Turn the music off/on if the student desires.

(b) Provide different sets of music to go with the app.

**Comment by ghost** [comment]
We need to be able to allow the app creator to provide music that the learner wants (without infringing anyone's copyright). This functionality would be valuable to everyone, but especially valuable to learners with learning difficulties like Asperger's, Autism, anxiety disorders, Dyslexia and other learning dfficulties.

**Comment by philiprbrenan** [comment]
I agree that the music is important.  It also signifies that we are in race mode vs normal play mode. In race mode the objective is to answer the questions quickly under pressure to simulate using the knowledge being disseminated in the noisy real world.  Thus if we were writing a fire fighting app we would want to hear fire, explosions, sirens, falling timbers etc., etc. rather than nice music.  So I have marked this issue as an 'enhancement' request and also as 'help wanted' if any-one out there cares to assist.

**Comment by philiprbrenan** [comment]
#43 delays when the music starts which might improve the situation.

**Comment by ghost** [comment]
I am not at all convinced by what you are saying in the email below :)

We are mostly going to be teaching sets of learning points and not simulating performance of a skill in the real word, although in some instances we may get close to performing a skill. For me the current music is so counterproductive that it would stop me using the app. 

If you say to me that some people like background music and others don’t then I do agree with that (and also agree that as you say, younger people expect a certain amount of jazz) - and then the discussion would be around the optimal type of music to have to encourage the learning. I very much doubt that the current music will help anyone to learn but I’m open to being proved wrong about that.

As the app is simulating fast-paced classroom drills in a digital format, it may help people get into a state of ‘flow’ or ‘peak performance’ in the same way that a good classroom drill does. So I’m wondering what music is most likely to be compatible with that state to get or encourage people into the same ‘flow’ state. Is it Gregorian chants... or no… I think my partner told me it’s Chamber Music that calms the brain most effectively. I don’t think the calm state is the same as the flow state but that a person needs to be calm in order to get into flow.

I could actually ask a friend who would know about this - in fact I will do that!

Jx
